lib/api/controllers/collectionController.js

```diff
@@ -296,6 +299,11 @@ class CollectionController extends NativeController {
       throw kerror.get('api', 'assert', 'invalid_argument', '"all", "stored", "realtime"');
     }
 
+    if (request.input.args.type) {
+      request.addDeprecation('auto-version',
```

Comment:
Unfortunately, the `auto-version` replacement currently only works for the documentation. And I don't know if we want to automatically make the replacement in the code itself. :thinking: 

I think that you'll need to put a hard-coded version number there. For now the next version will be `2.10.2`.

Reply:
IMHO we could look into the code to replace those strings as well

Reply:
Maybe we could have something more explicit like `{auto-version}`

Reply:
Could be a lead. In that case, I would go full explicit, with something like: `release-tools:auto-version` to make sure that we can't mistake this with any other string.

Reply:
I'm not fund of modifying strings within the code, how can we know that we would **never** write `release:auto-version` in a comment or a log?